server/routes/downloadReportConfig/nomis.test.ts

```diff
@@ -211,7 +211,7 @@ Question ID,Question sequence,Question list sequence,Question,Allows multiple an
               expiryDate: new Date(2022, 7, 20),
             },
             {
-              prisonerRole: 'LEAD',
```

Comment:
I'm guessing this may have stand for "leader"? But I don't see this code in [the generated prisoner roles](https://github.com/ministryofjustice/hmpps-incident-reporting/pull/188/commits/58e941489246144443fa7d4cf5d12dc1be718c02#diff-7cb234be6aab76e9d83dfaa2b0b7bb2c1dfb94e88661f8e227d1cbd0bd2e7bec).

Reply:
i must've buggered this one up. no idea where i got it from.